doc/ChangeLog

```diff
@@ -1,3 +1,10 @@
+28 October 2024: Jeroen
+	- Merge #404: Introducing Sphinx substitution in code blocks.
+	  As well as other fixes with Sphinx build.
+	- Merge #391: Update Copyright lines in help output
+	- Merge #395: Explain zonefile example better
+	- Merge #394: Fix doc path (fixes "Edit on GitHub" button in the docs)
```

Comment:
```suggestion
	- Merge #394: Fix doc path (fixes "Edit on GitHub" button in the docs)
	- Many thanks to Melroy van den Berg for the documentation improvements and fixes in PRs #391, #394, #395 and #404
```

Reply:
Haha thanks for that. It's not necessary needed. I was more confused about how the changelog worked with NLnet.

Reply:
Ps. I also notice there was a Credits file as well..